### ISSUE TITLE
[Bug] Favorites from Sankaku don't load in app after a period of time

### ISSUE THREAD

**Issue by Noname191** [NO-ob/LoliSnatcher_Droid#69]
Upon opening in the browser it links straight to the image. However when in the app, the image is only shown if it was favorited within the last day. 

![20210927_121751](https://user-images.githubusercontent.com/91490971/134955768-5b879b2f-b2b5-4b49-9f64-1ebb0c79f4f1.jpg)
![20210927_121819](https://user-images.githubusercontent.com/91490971/134955787-e357ff63-0613-4397-9315-a2efc40bf7a9.jpg)

**Comment by NANI-SORE** [comment]
Yeah, we knew about this problem from the start and I think there even was a disclaimer about it in one of the changelogs.
Sankaku is probably the only supported site which has expirable links to thumbnails/media files.
Links to post don't expire though, that's why you can open them just fine in the browser.

To get around this we would have to code logic exclusive to Sankaku which fetched new links when old ones expire.
For now it's a low priority issue (at least for me)

**Comment by Noname191** [comment]
I read through changelogs and things to see if I could find something about it but I must have missed it. Thanks for the reply.

**Comment by Anon1337Elite** [comment]
> Will be fixed in next update Settings > Database > Update Sankaku Urls

Same issue with Idol Senkaku please fix it.

https://github.com/NO-ob/LoliSnatcher_Droid/issues/285
